fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/logical/MergeLimits.java

```diff
@@ -43,11 +42,11 @@
 public class MergeLimits extends OneRewriteRuleFactory {
     @Override
     public Rule build() {
-        return logicalLimit(logicalLimit()).whenNot(Limit::hasValidOffset).then(upperLimit -> {
+        return logicalLimit(logicalLimit()).then(upperLimit -> {
             LogicalLimit<? extends Plan> bottomLimit = upperLimit.child();
             return new LogicalLimit<>(
                     Math.min(upperLimit.getLimit(), bottomLimit.getLimit()),
-                    bottomLimit.getOffset(),
+                    bottomLimit.getOffset() + upperLimit.getOffset(),
                     bottomLimit.child()
```

Comment:
only sum offset is not right
consider this scene:
before merge:
limit(limit = 7, offset = 5)
+-- limit(limit = 10, offset = 5)

after bottom limit, we get tuple from 6 to 15
after top limit, we get tuple from 11 to 15

so, after merge, we should get:
limit(limit = 5, offset = 10)

Reply:
Thanks for your kind reminder